webpack/localization-plugin-5/src/test/__snapshots__/LocalizedRuntime.test.ts.snap

```diff
@@ -0,0 +1,549 @@
+}
+`;
+
+exports[`LocalizationPlugin Handles async localized chunks (minified): Errors 1`] = `Array []`;
+
+exports[`LocalizationPlugin Handles async localized chunks (minified): Warnings 1`] = `Array []`;
+
+exports[`LocalizationPlugin Handles async localized chunks (unminified): Content 1`] = `
+Object {
+  "/release/chunks/async-en-us.js": "\\"use strict\\";
+(self[\\"webpackChunk\\"] = self[\\"webpackChunk\\"] || []).push([[931],{
+
+/***/ 222:
+/***/ ((__unused_webpack_module, __webpack_exports__, __webpack_require__) => {
+
+// ESM COMPAT FLAG
+__webpack_require__.r(__webpack_exports__);
+
+;// CONCATENATED MODULE: ./a/strings1.resjson
+const strings = {\\"test\\":\\"blah\\"};
+/* harmony default export */ const strings1_resjson = (strings);
+;// CONCATENATED MODULE: ./a/strings2.resjson
+const strings2_resjson_strings = {\\"another\\":\\"something else\\"};
+/* harmony default export */ const strings2_resjson = (strings2_resjson_strings);
+;// CONCATENATED MODULE: ./a/async.js
+  console.log(strings1_resjson.test, strings2_resjson.another);
+
+/***/ })
+
+}]);",
+  "/release/chunks/async-foo.js": "\\"use strict\\";
+(self[\\"webpackChunk\\"] = self[\\"webpackChunk\\"] || []).push([[931],{
+
+/***/ 222:
+/***/ ((__unused_webpack_module, __webpack_exports__, __webpack_require__) => {
+
+// ESM COMPAT FLAG
+__webpack_require__.r(__webpack_exports__);
+
+;// CONCATENATED MODULE: ./a/strings1.resjson
+const strings = {\\"test\\":\\"baz\\"};
```

Comment:
Seems like we should recommend people set the loader type to `javascript/esm` to avoid the `JSON.parse(...)` thing.

Reply:
The reason for the `JSON.parse` thing is that various perf tests were conducted that determined that `JSON.parse` is actually faster for large objects than writing out the object literal in JavaScript.

Reply:
Huh. Weird.